OptionsTable.md

```diff
@@ -6,7 +6,7 @@ Most (except for `-h` and `-prop`) of the options below can be passed to a SAVE
 | parallel | parallel-mode | Whether to enable parallel mode | false |
 | t | threads | Number of threads | 1 |
 | d | debug | Turn on debug logging | false |
-| q | quiet | Do not log anything | false |
+| q | quiet | Log only test results (if `--result-output` is `stdout`) without any warnings (only with critical errors) | false |
```

Comment:
I think it's different now, from what's in json

Reply:
updated, thanks